scripts/ctool

```diff
@@ -965,9 +965,23 @@ git_import() {
 
 
 delete_container() {
-    lxc delete --force "$1"
+    local rc=""
+    for x in 1 2 3 4 5; do
+        # this may fail on ZFS backed systems; let's retry a few
+        # times before reporting error
+        lxc delete --force "$1" 2>/dev/null
```

Comment:
Do we really want to discard the output from the `delete`? This could make debugging failures more difficult. If you're worried about a bunch of noise (which is a valid concern, of course), we could potentially not do the redirect when x=5, so we get the error output once for things which don't go away within 5 seconds?

(Sorry, should have caught this first time around.)

Reply:
You're right; I'll leave it there.  It's a big log in jenkins so not really so user-facing.